### ISSUE TITLE
fix: AppBlocObserver has an error on onError method

### ISSUE THREAD

**Issue by kirkcharlesniv** [VeryGoodOpenSource/very_good_cli#110]
**Description**
When upgrading the dependencies of `bloc`, and `flutter_bloc`, an error will occur on AppBlocObserver Line 20.

```
'AppBlocObserver.onError' ('void Function(Bloc<dynamic, dynamic>, Object, StackTrace)') isn't a valid override of 'BlocObserver.onError' ('void Function(BlocBase<dynamic>, Object, StackTrace)') .dart(invalid_override)
```
**Steps To Reproduce**

1. Create project using very_good cli 0.1.5
2. Update `bloc: ^7.0.0-nullsafety.3` to `bloc: ^7.0.0`, and `flutter_bloc: ^7.0.0-nullsafety.5` to `flutter_bloc: ^7.0.0`
3. Run flutter pub get

**Expected Behavior**
No errors

**Screenshots**
![Screen Shot 2021-03-18 at 8 58 41 PM](https://user-images.githubusercontent.com/19856377/111629885-b9148380-882c-11eb-8b84-a7cd45b2a4fd.png)

**Comment by felangel** [comment]
Hi @kirkcharlesniv 👋 
Thanks for opening an issue!

In v7.0.0 `onError` takes a `BlocBase`:

```dart
class MyBlocObserver extends BlocObserver {
  @override
  void onCreate(BlocBase bloc) {...}

  @override
  void onEvent(Bloc bloc, Object event) {...}

  @override
  void onChange(BlocBase bloc, Object? event) {...}

  @override
  void onTransition(Bloc bloc, Transition transition) {...}

  @override
  void onError(BlocBase bloc, Object error, StackTrace stackTrace) {...}

  @override
  void onClose(BlocBase bloc) {...}
}
```

For more information, you can refer to the [bloc migration guide](https://bloclibrary.dev/#/migration?id=v700). We will have a new release of very_good_cli that upgrades to v7.0.0 very soon 😄 

Hope that helps 👍

**Comment by felangel** [comment]
This has been addressed in #111 and published in v0.1.6 🎉
